dsl/camel-yaml-dsl/camel-yaml-dsl/src/test/groovy/org/apache/camel/dsl/yaml/ThrottleTest.groovy

```diff
@@ -53,7 +55,9 @@ class ThrottleTest extends YamlTestSupport {
                           - throttle: 
                               expression: 
                                 constant: "5"
-                              executor-service: "myExecutor"
+                              throttlerConfig:
+                                totalRequestsConfig:
+                                   executor-service: "myExecutor"
```

Comment:
I believe these changes should give an idea of the impacts on the YAML code for the throttler (c/c @oscerd, @davsclaus, @jono-morris).